src/wp-includes/kses.php

```diff
@@ -1311,6 +1319,54 @@ function wp_kses_attr_check( &$name, &$value, &$whole, $vless, $element, $allowe
 	return true;
 }
 
+/**
+ * If an attribute name represents a custom data attribute, return the
```

Comment:
Adds a summary so the details are all included in the description section of the developer docs ([see `wp_kses()` as an example](https://developer.wordpress.org/reference/functions/wp_kses/)).

```suggestion
 * Convert attribute to JavaScript `dataset` form if allowed.
 *
 * If an attribute name represents a custom data attribute, return the
```

Reply:
@peterwilsoncc are you recommending a different wording of the summary, or worried that the existing summary won't appear in the docs? The existing summary is up to two lines at the start of the docblock, so this will appear as-written in the patch.

For example, `class_name_updates_to_attributes_updates()` also has a two-line summary.

<img width="628" alt="Screenshot 2024-05-22 at 4 40 01 PM" src="https://github.com/WordPress/wordpress-develop/assets/5431237/9455b62d-7929-4e15-884f-60aa46cf8b60">

<img width="1022" alt="Screenshot 2024-05-22 at 4 39 33 PM" src="https://github.com/WordPress/wordpress-develop/assets/5431237/6e49b5fb-671d-4a80-8645-892dcb1261b9">

<img width="1019" alt="Screenshot 2024-05-22 at 4 39 38 PM" src="https://github.com/WordPress/wordpress-develop/assets/5431237/43ff2c98-a041-4bac-ad6e-c703984d0210">

Reply:
I'm mainly suggesting different, shorter, phrasing.

I thought the summaries were intended to be a one-liner but just re-read the [doc standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#docblock-formatting) and it turns out I was mistaken.

I'll leave this up to you.

Reply:
Reworded in dc859f19a5. It may take some more rewording 🤷‍♂️